src/Microsoft.Health.Fhir.Shared.Api.UnitTests/Features/SMART/SmartClinicalScopesMiddlewareTests.cs

```diff
@@ -70,18 +70,22 @@ public async Task GivenSmartScope_WhenInvoked_ThenScopeParsedandAddedtoContext(s
 
             var fhirUserClaim = new Claim(authorizationConfiguration.FhirUserClaim, "https://fhirServer/Patient/foo");
             var rolesClaim = new Claim(authorizationConfiguration.RolesClaim, "smartUser");
-            var scopesClaim = new Claim(authorizationConfiguration.ScopesClaim, scopes);
-            var claimsIdentity = new ClaimsIdentity(new List<Claim>() { scopesClaim, rolesClaim, fhirUserClaim });
-            var expectedPrincipal = new ClaimsPrincipal(claimsIdentity);
 
-            httpContext.User = expectedPrincipal;
-            fhirRequestContext.Principal = expectedPrincipal;
+            foreach (string singleClaim in authorizationConfiguration.ScopesClaim)
```

Comment:
This is a good change, where each of the different scopes are tested separately, but we also need some new tests where the different scopes are spread across the multiple claims.

For example, claim scope = "patient.Patient.read" and role claim = practitioner.Observation.write"  In the end both are picked up and applied.